app/controllers/hub/clients_controller.rb

```diff
@@ -105,8 +99,6 @@ def update_take_action
     end
 
     def unlock
-      raise CanCan::AccessDenied unless current_user.admin? || current_user.org_lead? || current_user.site_coordinator?
```

Comment:
this is now specified in ability.rb, functionality is the same with the addition that they will also see this page if client has archived intake